lib/animation.dart

```diff
@@ -196,6 +201,11 @@ class Animation {
     return loop ? false : (isLastFrame && clock >= currentFrame.stepTime);
   }
 
+  /// Register the method to be trigged when an animation is complete.
+  void addOnCompleteAnimation(OnCompleteAnimation onCompleteAnimation) {
```

Comment:
I think it's weird to name it add when it in fact replace. This might break the user expectations when they call this twice. In fact though, instead of simply having a plain setter, we can make the field public. we can always add getters & setters later if needed w/o breaking change.

Reply:
Ok I can change to be that way just and public var onCompleteAnimation, or I could change to be a list of OnCompletAnimation so the add can stack the events and works in an improved way, please let me know your opinion on this.

Reply:
I think just making it public is good enough :) a list might be overkill for now.

Reply:
Ok will do it ;)